src/components/calcite-block/calcite-block.tsx

```diff
@@ -152,6 +152,14 @@ export class CalciteBlock {
       </header>
     );
 
+    let hasControl = false;
+    const hasContent = Array.from(el.children).some((child) => {
+      const isControl = child.slot === SLOTS.control;
+      hasControl = hasControl || isControl;
```

Comment:
pebble: array.prototype.some stops running on items in the array once it returns a true state.  It possible there'd be a bug where there is a control but it isn't found because `.some` stopped checking remaining items?

Reply:
Doh! Good catch. 🛠